rmgpy/kinetics/arrhenius.pyx

```diff
@@ -650,10 +650,9 @@ cdef class ArrheniusBM(KineticsModel):
         A = np.exp(lnA)
 
         # fill in parameters
-        if len(rxns[0].reactants) == 1:
-            self.A = (A, 's^-1')
-        elif len(rxns[0].reactants) == 2:
-            self.A = (A, 'm^3/(mol*s)')
+        A_units = {1: 's^-1', 2: 'm^3/(mol*s)', 3: 'm^6/(mol^2*s)'}
```

Comment:
You could make this a list because the keys are integers in increasing order (make `''` the zeroth element), but I don't have a strong preference if you want to keep it this way.

Reply:
I changed it to a list